orc8r/cloud/go/swagger/spec.go

```diff
@@ -36,3 +36,12 @@ type TagDefinition struct {
 	Description string
 	Name        string
 }
+
+// MarshalBinary marshals the Spec structure into a byte form.
```

Comment:
- `MarshalBinary marshals the spec to bytes`
- Also you can optionally leave the doc out, since it's an iface impl. Up to you